payment_bitcoin/tests/test_payment_bitcoin.py

```diff
@@ -0,0 +1,230 @@
+import codecs
+from datetime import datetime
+from datetime import timedelta as td
+from hashlib import sha256
+from ..models.bitcoin import validate_bitcoin_address, check_received
+
+DIGITS58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
+CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l"
```

Comment:
Looks confusing. Shouldn't this be alphabetically ordered?

Reply:
I checked and it is redundant code I'll remove it.